### DISCUSSION TITLE
Commonsense Knowledge, Symbolic Knowledge Distillation paper: What is "generation"?

### DISCUSSION THREAD

**Discussion by mayanks43** [eduardoblanco/csc696H-Spring24#12] [Q&A]
"Some CKGs are automatically constructed: TransOMCS (Zhang et al., 2020a) extracts 18.48M tuples from syntactic parses and CausalBank (Li et al., 2020) extracts 314M cause-effect pairs by pattern-matching. In contrast, we generate commonsense." Is this true? What do they mean by "generating"? Are they "generating" commonsense? Why are other methods not "generating"?

**Comment by Abby-OGV** [comment]
TransOMCS uses rules to get the cause-effect pairs from existing text. So, the data already exists and the information needed (the pairs) is just being extracted.
$`ATOMIC^{10X}`$, on the other hand, is prompting a model for an answer.

**Comment by eduardoblanco** [comment]
I think running a rule on the web and getting the matches would not be considered generating. It is extracting.
